static/app.js

```diff
@@ -733,14 +733,16 @@ $(function() {
     });
 
     $('#sort_by_number').on('click', function() {
-        sort_function = function(l, r) {
-            let c1 = new Card(l);
-            let c2 = new Card(r);
-            if (l.value != r.value) {
-                return l.value - r.value;
+        function sort_func_by_number(suite, val, l, r) {
+            l = normalize_card_val(l, suite, val);
+            r = normalize_card_val(r, suite, val);
```

Comment:
Not a big deal, but probably worthwhile to consider using destructuring assignments to improve readability.
For example:
```
[l_suite, l_value] = normalize_card_val(l, suite, val);
[r_suite, r_value] = normalize_card_val(r, suite, val);
```
Then I do not need to check func `sort_func_by_number` 's return values.
* Note that this feature is supported after JS 1.7

Reply:
Good point!